airqo-platform/AirQo-frontend
# [WebSite] Data credibility

#### Summary of Changes (What does this PR do?)
- Ensures credibility of the data showcased on the maps of the website by updating website cache with the most recent device events/readings. 
- Old/ retired devices are removed too.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch this branch locally.
(for detailed setup follow this [PR](https://github.com/airqo-platform/AirQo-frontend/pull/208))
* Set `.env` with respective values ([slack](https://airqo.slack.com/archives/GTHGHCB4G/p1618994366002700))
* Build app `docker-compose build`
* Run App `docker-compose up`
* Check the website map (http://localhost:8000/). Will contain devices that last pushed data 4-6 months ago.
* Manually trigger cache update using this endpoint `POST http://localhost:8000/Apis/airqoPlacesJobUpdate`. sample Data 

```
{
    "api": "random_string"
}
```
* Reload the website page (http://localhost:8000/), all devices should have last pushed <= a month

#### What are the relevant tickets?
- [PLAT-504](https://airqoteam.atlassian.net/browse/PLAT-504)

#### Screenshots (optional)
